mmselfsup/datasets/pipelines/transforms.py

```diff
@@ -482,3 +482,113 @@ def __repr__(self):
         repr_str += f'threshold = {self.threshold}, '
         repr_str += f'prob = {self.prob}'
         return repr_str
+
+
+@PIPELINES.register_module()
+class MaskfeatMaskGenerator(object):
```

Comment:
How about use `MaskFeatureMaskGenerator`? @fangyixiao18 
```suggestion
class MaskFeatMaskGenerator(object):
```

Reply:
I think it is better to use `MaskFeatMaskGenerator`，consisting with algorithm name in our implementation